lib/model/txproposal.js

```diff
@@ -71,6 +71,7 @@ TxProposal.create = function(opts) {
   x.excludeUnconfirmedUtxos = opts.excludeUnconfirmedUtxos;
   x.proposalSignaturePubKey = opts.proposalSignaturePubKey;
   x.proposalSignaturePubKeySig = opts.proposalSignaturePubKeySig;
+  x.derivationStrategy = opts.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP45;
```

Comment:
That is correct. `AddressManager` is an exception as it is the only structure not tied to an API and so it doesn't have to default to BIP45. But anyways, see my comment above.